### PR TITLE
fixed Issue #1443

### PR DESCRIPTION
**Pull request by brett-schneider**

### CODE REVIEW

**Comment by brett-schneider** [comment]
also added endpoint
[get-lists-ownerships](https://developer.twitter.com/en/docs/twitter-api/v1/accounts-and-users/create-manage-lists/api-reference/get-lists-ownerships)

File: tweepy/api.py [hunk 1]
```diff
@@ -1034,6 +1034,21 @@ def lists_memberships(self):
...
+        """
+        return bind_api(
+            api=self,
+            path='/lists/ownerships.json',
+            payload_type='list', payload_list=True,
+            allowed_param=['screen_name', 'user_id',
+                           'cursor', 'count'],
+            require_auth=True
+        )
+
```
Comment (Harmon758) [inline tweepy/api.py:1051]: This should be part of a separate pull request, which should also add documentation for the method.

File: tweepy/api.py [hunk 2]
```diff
@@ -1158,14 +1173,14 @@ def _remove_list_members(self):
...
-                            'cursor'
+                            'cursor', 'count'
         """
         return bind_api(
             api=self,
             path='/lists/members.json',
             payload_type='user', payload_list=True,
             allowed_param=['owner_screen_name', 'slug', 'list_id', 'owner_id',
-                           'cursor']
+                           'cursor', 'count']
```
Comment (Harmon758) [inline tweepy/api.py:1183]: This should also be part of another separate pull request, which should also add documentation for the new allowed parameter.

File: tweepy/models.py [hunk 3]
```diff
@@ -196,17 +196,22 @@ def unfollow(self):
         self.following = False
 
     def lists_memberships(self, *args, **kwargs):
-        return self._api.lists_memberships(user=self.screen_name,
+        return self._api.lists_memberships(user_id=self.id,
+                                           *args,
+                                           **kwargs)
+
+    def lists_ownerships(self, *args, **kwargs):
+        return self._api.lists_ownerships(user_id=self.id,
```
Comment (Harmon758) [inline tweepy/models.py:204]: This should be part of the separate pull request that adds `API.lists_ownerships`.

**Review by Harmon758 — CHANGES_REQUESTED** [review]

**Comment by Harmon758** [comment]
Since this is the last PR I want to include for v3.10, I've gone ahead and removed the extraneous changes myself.
Feel free to open separate PRs for them.

**Review by Harmon758 — APPROVED** [review]

**Pull request merged**